teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -327,139 +299,211 @@ $ kubectl describe task.tekton.dev/golang-lint
 Name:         golang-lint
 Namespace:    default
 Labels:       app.kubernetes.io/version=0.3
-              hub.tekton.dev/catalog=tekton-community
+              hub.tekton.dev/catalog=tekton
+... 
+```
+
+However, users would also need the source Org in case multiple Catalogs have the same name. 
+
+We propose deprecating the `--from` field and replacing it with `--catalog` and `--org`. This should add labels to 
+the resource indicating the source Catalog (`hub.tekton.dev/catalog`), the source Org (`hub.tekton.dev/org`) and the 
+support tier (`hub.tekton.dev/type`).
+
+```shell
+# Buildpacks Community Catalog
+
+$ tkn hub install task buildpacks --version 0.3 --catalog buildpacks --org buildpacks
+
+Task buildpacks(0.3) installed in default namespace
+
+$ kubectl describe task.tekton.dev/buildpacks
+
+Name:         buildpacks
+Namespace:    default
+Labels:       app.kubernetes.io/version=0.3
+              hub.tekton.dev/catalog=buildpacks
+              hub.tekton.dev/org=buildpacks
+              hub.tekton.dev/type=community            
 ...
  
-# Official Catalog
+# Git Official Catalog
 
-$ tkn hub install task golang-fuzz --version 0.1 --from tekton-official
+$ tkn hub install task git-clone --version 0.7 --catalog git --org tektoncd-catalog
 
-Task golang-fuzz(0.1) installed in default namespace
+Task git-clone(0.7) installed in default namespace
 
-$ kubectl describe task.tekton.dev/golang-fuzz
+$ kubectl describe task.tekton.dev/git-clone
 
-Name:         golang-fuzz
+Name:         git-clone
 Namespace:    default
-Labels:       app.kubernetes.io/version=0.1
-              hub.tekton.dev/catalog=tekton-official
+Labels:       app.kubernetes.io/version=0.7
+              hub.tekton.dev/catalog=git
+              hub.tekton.dev/org=tektoncd-catalog
+              hub.tekton.dev/type=official
 ...
 ```
 
-#### Community and Official Catalogs in the Cluster
+#### Cluster
 
-When resources are installed in a cluster, without using the CLI, 
-it may be difficult to identify which Tekton Catalog it came from
-because they won't have the labels added by the CLI.
+When resources are installed in a cluster, without using the CLI, it may be difficult to identify which Tekton Catalog
+it came from because they won't have the labels added by the CLI.
 
-To make it easy for users to identify the source Catalog, we propose 
-adding two annotations:
-* `tekton.dev/catalog` with the three part domain unique identifier
+To make it easy for users to identify the source Catalog from the Hub, we propose adding two annotations:
+* `tekton.dev/catalog` with the three-part domain unique identifier: `<catalog-tier>.<catalog-org>.<catalog-name>`
```

Comment:
I get the rationale for having the tekon.dev/catalog label but why do we need the catalog tier as part of the identifier and why can't be its own label?